pom.xml

```diff
@@ -122,6 +123,12 @@
             <artifactId>credentials-binding</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness-htmlunit</artifactId>
+            <version>225.v7d540e18a_8f4</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Should be in `dependencyManagement` (and `scope` can be omitted).

Do we need both this _and_ the `jenkins-test-harness` bump?

Reply:
(The harness did not have the htmlunit bump. Both needed bumping to get the tests to pass. Going the long way as #250 did, is indeed the proper upgrade. :+1: )